Assets/unZENity-VR/Tests/EditorTests/FilesCheck/FilesCheck.asmdef

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What is this file? Just for my interest. :-)

Reply:
Only an assembly reference for the Testrunner.
So that it knows where the tests are.